app/dashboard/layout.tsx

```diff
@@ -20,7 +20,7 @@ export default async function DashboardLayout({
   return (
     <div className="w-full">
       <SideNav />
-      <main className="min-h-screen 2xl:ml-80 ml-64 2xl:px-44 px-14 2xl:py-16 py-10">{children}</main>
+      <main className="min-h-screen 2xl:ml-70 ml-64 2xl:px-44 px-14 2xl:py-16 py-10">{children}</main>
```

Comment:
The margin left needs to be the same width as the sidenav its what keeps anything from going under the side nav if you wanna decrease space after sidenav you could change the px padding